src/fragmentarium/ui/fragment/TeiExport.tsx

```diff
@@ -0,0 +1,161 @@
+import { Fragment } from 'fragmentarium/domain/fragment'
+import { AbstractLine } from 'transliteration/domain/abstract-line'
+import { TextLine } from 'transliteration/domain/text-line'
```

Comment:
Remove unused imports.

Reply:
left in for the purpose of demonstration, will hopefully be used when type issue is solved